### ISSUE TITLE
Run problem on AWS EC2

### ISSUE THREAD

**Issue by beratyildizulak** [freqtrade/ftui#5]
hi,

firstly thanks for your project to share.

I try running ftui on aws ec2 ubuntu. I connected the server via ssh. But I got error as below :
```
~/ftui$ ftui -y ftui.yaml
Traceback (most recent call last):
  File "/home/ubuntu/ftui/.venv/bin/ftui", line 5, in <module>
    from ftui.ftui import main
  File "/home/ubuntu/ftui/ftui/ftui.py", line 66, in <module>
    class FreqText(App):
  File "/home/ubuntu/ftui/.venv/lib/python3.10/site-packages/textual/message_pump.py", line 107, in __new__
    class_obj = super().__new__(cls, name, bases, class_dict, **kwargs)
  File "/home/ubuntu/ftui/.venv/lib/python3.10/site-packages/textual/app.py", line 694, in __init_subclass__
    raise ValueError(
ValueError: MODES should contain a Screen type or callable, not an instance (got instance of DashboardScreen for 'dashboard')
```
Do you have any comment?

**Comment by xmatthias** [comment]
well i can reproduce this easliy (apparently independent of the environment - at least it appears that way at first).

What's causing it is not immediately clear - but we'll take a look for sure

**Comment by xmatthias** [comment]
@beratyildizulak can you please try reinstalling the latest version? 
i've just released version 0.1.3 - aimed to provide a quick-fix to your issue.


@froggleston i've had to pin textual to <0.77 for now.
it's caused by the following line in the changelog to 0.77 - and will need investigation eventually - though i lack experience with textual per se to be of great help here.

> Disallowed Screen instances in App.SCREENS and App.MODES

[source](https://github.com/Textualize/textual/releases/tag/v0.77.0)

**Comment by froggleston** [comment]
The underlying cause is described here: https://github.com/Textualize/textual/issues/4893

I'm trying to work out how to refactor the app to take this into account, but @xmatthias 's solution will work for now.

**Comment by froggleston** [comment]
All issues addressed, will be fixed next release.

**Comment by beratyildizulak** [comment]
Sorry for late reply. Thanks for your interest. The problem is fixed with "commit 2ceaa60a91c6c3bc0c1e69dc536497ccb1a1e9c0". I am closing the case.

**Comment by froggleston** [comment]
Thanks for letting us know! All fixes are in the latest release: https://github.com/freqtrade/ftui/releases/tag/0.1.4
